### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by BoAnd**
Simple README.md file with build info. Please add more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Sangbogen
+========
+
+This is the current sangbog in the F-Klub
+
+Latex Environment
+-------------
+For apt systems:
+1. In a shell
+  - `git clone https://github.com/f-klubben/stregsystemet.git`
```
Comment (falkecarlsen) [inline README.md:10]: This should be sangbog.git

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2. Install needed packages:
+  - `sudo apt install texlive-latex-extra psutils texlive-lang-european`
```
Comment (falkecarlsen) [inline README.md:12]: Maybe a note on which distro these packages are available on?

**Review by falkecarlsen — CHANGES_REQUESTED** [review]
Nice, thanks for writing it - smaller issues though

File: README.md (hunk 2 above)
Comment (falkecarlsen) [reply README.md:12]: Ah, you wrote that - my bad

File: README.md (hunk 1 above)
Comment (BoAnd) [reply README.md]: Ups, fixed.

I hope we will get a lot of new songs now.

**Review by falkecarlsen — APPROVED** [review]

**Pull request merged**